publicSite/views.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.http import HttpResponseRedirect, HttpResponse
+
+from django.core.urlresolvers import reverse
+from django.shortcuts import render
+
+from publicSite.models import Helper, Helpee, Demographic
+
+# Create your views here.
+def index(request):
+    return render(request, 'index.html', { 'launched': False })
```

Comment:
Currently, this `launched` flag causes display of some pre-launch text so people can't sign up as helper / helpee.